notification-portlet-webapp/src/main/resources/META-INF/persistence.xml

```diff
@@ -54,6 +59,9 @@
                  hsqldb we need this to be update. Users *should* do a schema update before switching to new versions, or
                  this should hopefully not do something negative if it ever does update the schema (typically doesn't). -->
             <property name="hibernate.hbm2ddl.auto" value="update" />
+            
+            <!-- Dialect needed for POSTGRES to work -->
+            <property name="hibernate.dialect" value="${environment.build.hibernate.dialect}"/>
```

Comment:
Discussed with @chasegawa.  He said he tested with ${hibernate.dialect} and that worked, so don't need to mess around with the items in my suggestion.  Just waiting on updated PR.